melodic/distribution.yaml

```diff
@@ -2069,6 +2069,17 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  cyglidar_d1:
+    doc:
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.2
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.2
```

Comment:
The `release` section should be updated by Bloom, and the URL should point to a release repository, not the source repository. You should add a `source` section instead.

If you'd like to officially release your package to be built on the buildfarm, please take a look at the [First Time Release](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease#Releasing_Your_Packages) tutorial.

Reply:
Thanks for the guidance. Just requested to pull via Bloom again.